client/lib/feed-stream-store/index.js

```diff
@@ -140,6 +140,19 @@ function getStoreForFeatured( storeId ) {
 	} );
 }
 
+function getStoreForWarmstart( storeId ) {
```

Comment:
see other comment, but warmStart is an internal name. we should probably call this what is is, recommended posts.